packages/dds/sequence/src/test/fuzz/intervalCollection.fuzz.spec.ts

```diff
@@ -50,7 +50,7 @@ describe("IntervalCollection with stashing", () => {
 			stashableClientProbability: 0.2,
 		},
 		// AB#7220
-		skip: [22, 56, 86],
+		skip: [4, 11, 18, 29, 35, 41, 47, 53, 65, 66, 73, 76, 86, 93, 96, 99],
```

Comment:
Both interval collection with stashing and shared string with stashing have a lot of failing seeds - does this point to a larger problem than the existing bugs we have open?

Reply:
possibly. i think i'm ok with the skips in this pr, as we are going to fix these issue shortly after correct?

Reply:
Correct, I'm planning to look into them once I finish this work

Reply:
I think that's fine. One thing I like to do as a sniff test when I'm unsure if the changes I've made are correct is minimizing / writing a minimal repro for one of the skipped seeds as part of the PR that skips it. This is also convenient for filing bugs (you can just point at the skipped but checked in test).

Your changes in this PR are pretty safe though. Assuming the failures you get have call stacks which actually suggest eventual consistency issues and don't set off alarm bells based on the changes here, this seems unnecessary for this change.